src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -212,23 +212,28 @@ public function addMethodMappings(array $methodNames)
     /**
      * Enables annotation based constraint mapping.
      *
+     * @param bool $skipDoctrineAnnotations
+     *
      * @return $this
      */
-    public function enableAnnotationMapping(Reader $annotationReader = null)
+    public function enableAnnotationMapping(/* bool $skipDoctrineAnnotations = true */)
```

Comment:
Shouldn't it be `enableAnnotationMapping()` (without argument) for consistency with `disableAnnotationMapping()`?

Reply:
Currently, the method creates a Doctrine annotation reader if I don't pass anything. That's the behavior I want to deprecate, but I need to maintain it for BC. The parameter tells me that the caller wants the new behavior. We can remove the parameter in Symfony 6.

Reply:
we even don't need a docker image. Github Action can run Dockerfile directly (https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-docker-container-action)